XmlFormatter/src/Settings/Adapter/PropertyAdapter.cs

```diff
@@ -126,5 +126,41 @@ public ISettingScope GetSubScope(string name)
         {
             return null;
         }
+
+        /// <summary>
+        /// This method is not supported by this adapter
+        /// </summary>
+        /// <param name="name">The name of the scope to remove</param>
+        /// <returns>Always true</returns>
+        public bool RemoveSubScope(string name)
+        {
+            return true;
+        }
+
+        /// <summary>
+        /// Method is not supported by the adapter
+        /// </summary>
+        public void ClearSubScopes()
+        {
+            return;
+        }
+
+        /// <summary>
+        /// This method is not supported by this adapter
+        /// </summary>
+        /// <param name="name">The name of the setting to remove</param>
+        /// <returns>Always true</returns>
+        public bool RemoveSetting(string name)
+        {
+            return true;
+        }
+
+        /// <summary>
+        /// Method is not supported by the adapter
+        /// </summary>
+        public void RemoveSettings()
+        {
+            return;
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Remove this redundant jump.](https://app.codacy.com/manual/simonaberle/XmlFormatter/pullRequest?prid=5053278)